packages/flutter/lib/src/material/tooltip_theme.dart

```diff
@@ -67,6 +67,9 @@ class TooltipThemeData with Diagnosticable {
   ///
   /// If there is insufficient space to display the tooltip in the preferred
   /// direction, the tooltip will be displayed in the opposite direction.
+  ///
+  /// Applying `false` for your entire app is recommended
+  /// to avoid having a finger or cursor hide a tooltip.
   final bool? preferBelow;
```

Comment:
The style guide says to avoid "you":
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use-the-passive-voice-recommend-do-not-require-never-say-things-are-simple

Can just s/your/the/, I think.